source/features/highlight-collaborators-prs.tsx

```diff
@@ -0,0 +1,46 @@
+import './highlight-collaborators-prs.css';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getRepoGQL} from '../libs/utils';
+
+function buildQuery(): string {
+	return `
+		repository(${getRepoGQL()}) {
+			collaborators(first: 100) {
+				nodes { login }
+			}
+		}
+	`;
+}
+
+async function init(): Promise<false | void> {
+	if (!features.isRepoWithAccess()) {
+		return false;
+	}
+
+	const authors = select.all('.js-issue-row [data-hovercard-type="user"]');
+	if (authors.length === 0) {
+		return false;
+	}
+
+	const data = await api.v4(buildQuery());
+	const collaborators = data.repository.collaborators.nodes.map((node: AnyObject) => node.login);
```

Comment:
I'd rather see a `getCollaborators` function instead of a random `buildQuery`


Also the feature should work on the global PR list as well (`/pulls`) as suggested for this other feature https://github.com/sindresorhus/refined-github/pull/2310#discussion_r311644086

Reply:
> I'd rather see a `getCollaborators` function instead of a random `buildQuery`

:+1:, was just copied from other feature. Will fix.



> Also the feature should work on the global PR list as well (`/pulls`) as suggested for this other feature

That will make it an magnitude harder. Running this on an repo discussion list binds it to just one repo (one fetch). The global discussion list can be any repo; possible extending the query by 10 (page size) repo fetches.

Reply:
> That will make it an magnitude harder

It _is_ a bit harder, repos would have to be de-duplicated so they only appear once in each fetch. 

Perhaps this info is not as useful in those pages in the first place.